mavros/c0067f9/c0067f9.bug

```diff
@@ -69,4 +69,4 @@ fault-codes:
   - EVOLUTION:LIBRARY
 failure-codes:
   - SOFTWARE:CRASHING
-  - SYSTEM:LOSS-OF-CONTROL
+  - SYSTEM:UNRESPONSIVE
```

Comment:
Is this component used for users to issue commands to the drone?

Reply:
This bug should cause MAVROS to collapse entirely. Unresponsive looks appropriate to me.